semcore/core/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
+## [2.12.1] - 2023-12-14
+
+### Fixed
+
+- Types in `Component`.
```

Comment:
Looks like it is totally internal change that has no sense for our users

Reply:
So, what do you think we need to write? `Fixed internal types`? 
Because we need to add some note for create a new versioin

Reply:
I think it's better not to write it at all. We don't need new version to be published with internal change

Reply:
`Fixed internal types` added